### PR TITLE
Feat functions docs improvements

### PR DESCRIPTION
**Pull request by christyjacob4**
screencapture-localhost-2080-docs-functions-2022-02-20-17_11_01

### CODE REVIEW

**Comment by eldadfux** [comment]
The indentation in some of the code examples is wrong.

**Comment by eldadfux** [comment]
Maybe we can also use a friendlier examples with some usage of headers or env vars.

**Comment by eldadfux** [comment]
On the execute section, it would be nice to get iOS and Android examples.

File: app/views/docs/functions.phtml [hunk 1]
```diff
@@ -183,36 +130,147 @@ pub fn main(req: RequestValue) -> Response {
...
-</div>
-    
-<h3>Accessing your Local Appwrite Setup</h3>
-    
-<p>If you're running Appwrite locally and trying to connect to your local Appwrite instance from a cloud function, you'll have to use your local IP address instead of localhost. This is because if you use localhost, the container that runs the cloud function will try to connect to itself rather than your 'appwrite' container.</p>
+<p>Deployments that have been built successfully are marked as <b>ready</b> and can be activated and executed.</p>
 
 <div class="notice">
-    <h3>Note: </h3>
+    <h3>A note about build times</h3>
```
Comment (eldadfux) [inline app/views/docs/functions.phtml:273]:
```suggestion
    <h3>Build Times</h3>
```

**Review by eldadfux — CHANGES_REQUESTED** [review]

**Comment by christyjacob4** [comment]
> The indentation in some of the code examples is wrong.

Fixed 👍 

> Maybe we can also use a friendlier examples with some usage of headers or env vars.

For all the runtimes ? Or some specific ones only ?  @eldadfux 

> On the execute section, it would be nice to get iOS and Android examples.

Done 👍

**Comment by christyjacob4** [comment]
Please don't merge it yet. The example code for ruby and swift is still under works

**Review by eldadfux — APPROVED** [review]

**Comment by christyjacob4** [comment]
screencapture-localhost-2080-docs-functions-2022-02-23-20_51_02
Latest Screenshot

**Pull request merged**